data/states.js

```diff
@@ -0,0 +1,282 @@
+var stateData = [
```

Comment:
This list gets looped through in 3 places, two of which look up by USPS. I think I would prefer this to be an object that does {UPSP :{stateData}}. I will also change our data input to make sure the state abbr is stored with every event, but in the meantime we could have another mapping of Statename to state abbr, which would still be faster than looping through all 50 states.